lib/justified-columns.js

```diff
@@ -1,5 +1,14 @@
-JustifiedColumns = (function(window) {
-
+(function (root, factory) {
+    if ( typeof define === 'function' && define.amd ) {
```

Comment:
Not as familiar with the syntax to support multiple module types, but it looks like we could just declare `factory(root)` before the conditional and just pass or return it as needed